client/my-sites/checkout/checkout-thank-you-i2/common/render-thank-you-redesign-i2.tsx

```diff
@@ -0,0 +1,17 @@
+import { isBusiness } from '@automattic/calypso-products';
+import { ReceiptPurchase } from 'calypso/state/receipts/types';
+import ThankYouRedesignI2 from '..';
+
+export const shouldRenderThankYouRedesignI2 = ( purchases: ReceiptPurchase[] ) => {
+	for ( const purchase of purchases ) {
+		switch ( true ) {
+			case isBusiness( purchase ):
```

Comment:
We can drop the logic in as we go through the refactor.
If there are things we don't even know in the original logic, then we'll return false and let it default to the legacy component.